.magicignore

```diff
@@ -0,0 +1,5 @@
+.git
+.libs
+rock_tmp
```

Comment:
No, no ooc files there, so it is completely unnecessary for magic to look there. Actually, these folders should be excluded by default, I just haven't got around to that yet. I suggest we keep those in there for now. Excluding them will boost performance a bit.